cluster/charts/rook-ceph/values.yaml

```diff
@@ -321,3 +321,8 @@ discoverDaemonUdev:
 
 # Whether the OBC provisioner should watch on the operator namespace or not, if not the namespace of the cluster will be used
 enableOBCWatchOperatorNamespace: true
+
+# clusterNamespaces option allowes to create necessary resources in order to create Ceph clusters in the specified namespaces.
+# clusterNamesspaces:
+# - cluster-ns1
```

Comment:
For the example, how about add a single example for `rook-ceph`?

Reply:
I think that with these changes, we should have `rook-ceph` be the default set option, which will instruct helm to create the resources that will allow the cluster to run in the same namespace as the operator. If a user were to remove `rook-ceph` from the options here, attempts to create a custom Ceph resources in the same namespace as the operator should fail. Does this make sense @travis?

Reply:
Whether the operator is in the same namespace as a cluster doesn't seem like it would matter. The operator will run in whatever namespace is specified with the `helm install --namespace`, and the clusters can be created in the namespaces according to this list. If there are no namespaces in this list, then creating a cluster would always fail.

Reply:
I think we're on the same page here. It sounds like you're describing what I was suggesting is my expectation. It sounds like that is what happens with this PR. :+1: